superset-frontend/.eslintrc.js

```diff
@@ -292,6 +386,7 @@ module.exports = {
   ],
   rules: {
     'theme-colors/no-literal-colors': 'error',
+    'icons/no-fa-icons-usage': 'warn',
```

Comment:
Personally would much rather have a proper `error`, combinated with `# NOQA` type statements for all of the entries we need to fix. Stops the bleeding in its track, and it's pretty easy to `grep` for the related NOQA statements.